assets/js/form.js

```diff
@@ -0,0 +1,43 @@
+// whole form
+// view
+var FormView = Backbone.View.extend({
+  template: AppTemplates.form,
+
+  el: '#target',
+
+  initialize: function() {
+    this.render();
+  },
+
+  events: {
+    'click .submit': 'submitForm'
+  },
+
+  render: function() {
+    var html = this.template(this.model.toJSON());
+    this.$el.html(html);
+
+    return this;
+  },
+
+  submitForm: function() {
+    var _this = this;
+    this.$el.submit(function() {
```

Comment:
You won't want to submit your form again. You are already listening for the form to submit when you registered your event on line 13.